tests/unit/cluster/cluster-migrateslots.tcl

```diff
@@ -403,6 +403,34 @@ start_cluster 3 3 {tags {logreqres:skip external:skip cluster} overrides {cluste
     set 16382_slot_tag "{4oi}"
     set 16383_slot_tag "{6ZJ}"
 
+    test "Slot migration won't migrate the functions" {
+        assert_does_not_resync {
+            # R 2 load a function then trigger a slot migration to R 0
+            populate 1 "$16383_slot_tag:" 1000 -2
+            R 2 function load {#!lua name=test
+                server.register_function('test', function() return 'hello1' end)
+            }
```

Comment:
Do we also need to test loading a function during the incremental stream?

It should be handled (since the command has no slot) but probably a good test case to check in this test

Reply:
good idea, i added it.